edge-util/src/Microsoft.Azure.Devices.Edge.Util.Metrics/Microsoft.Azure.Devices.Edge.Util.Metrics.csproj

```diff
@@ -20,11 +20,14 @@
     <PackageReference Include="prometheus-net.AspNetCore" Version="4.2.0" />
 
     <!--
-      prometheus-net.AspNetCore takes a transitive dependency on
-      System.Test.Encodings.Web. Use an explicit reference here to override the
-      version and fix a security vulnerability.
+      prometheus-net.AspNetCore takes transitive dependencies on
+      System.Test.Encodings.Web, Microsoft.AspNetCore.Http, and
+      Microsoft.AspNetCore.Server.Kestrel.Core. Use an explicit
+      reference here to override the version and fix a security vulnerability.
       See https://github.com/dotnet/runtime/issues/49377#issuecomment-804930299.
     -->
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.34" />
+    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Core" Version="2.1.25" />
     <PackageReference Include="System.Text.Encodings.Web" Version="4.5.1" />
```

Comment:
Why not update the version of System.Text.Encodings.Web? Version 4.5.1 is pretty old now.

Reply:
Oh, nvm, our dependency on Microsoft.AspNetCore.Http and Microsoft.AspNetCore.Server.Kestrel.Core come through prometheus-net.AspNetCore, not System.Text.Encodings.Web.